recipe/meta.yaml

```diff
@@ -6,22 +6,27 @@ package:
   version: {{ version }}
```

Comment:
There is a new v0.8.0 with a new API https://pypi.org/project/perpetual/0.8.0/. I would update to the latest version because users can request it again and we'll do double work.